### ISSUE TITLE
Implement Single Column Page

### ISSUE THREAD

**Issue by nikkithayer** [UCLALibrary/ursus#254]
**ACCEPTANCE**
- [ ] Text matches font styles available here: https://projects.invisionapp.com/d/main#/console/16734276/350465649/inspect
- [ ] Includes masthead - best way to curate and rotate mastheads is TBD. Mark proposed a set of 6 that randomly load.
- [ ] Works on mobile

Blocked by #269

**DETAILS**
* Assets are available here: https://projects.invisionapp.com/d/main#/console/16734276/350465646/inspect
* Colors and style guide are available here: 
https://projects.invisionapp.com/d/main#/console/16734276/350465650/inspect
* Full brand guidelines at:
http://brand.ucla.edu/identity/colors

**MOCK-UP**
![image](https://user-images.githubusercontent.com/1448246/53847469-b3a9b080-3f65-11e9-8a2c-042ff62f11a0.png)
![image](https://user-images.githubusercontent.com/1448246/53851120-6b44bf80-3f72-11e9-9215-5c5784b10389.png)

**Comment by mark-dce** [comment]
This is probably a duplicate of #269 - Implement an about page.  The other ticket probably has more concrete requirements.  We should review this after #269 is done and see if there is any remaining work to be done, otherwise de-point this and close it as a duplicate.
